deployments/mage/magefile.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2021 ZUP IT SERVICOS EM TECNOLOGIA E INOVACAO SA
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build mage
+// +build mage
+
+// Horusec-Plaform mage file.
+package main
+
+import (
+	"fmt"
+	"github.com/magefile/mage/mg"
+	"github.com/magefile/mage/sh"
+	"os"
+	// mage:import
+	_ "github.com/ZupIT/horusec-devkit/pkg/utils/mageutils"
+)
+
+//UpdatePackageJson updates packge.json lock horusec version
+func UpdatePackageJson(version string) error {
+	if err := sh.RunV("npm", "install", "-g", "json"); err != nil {
+		return err
+	}
+	return sh.RunV("json", "-I", "-f", "./manager/package.json", "-e", `'this.version="`+version+`"'`)
+}
+
+func UpdateHorusecVersionInProject(actualVersion, releaseVersion string) error {
+	dir, _ := os.Getwd()
```

Comment:
I think that would be good check this error

Reply:
this was a debug test gointo to erase